index.js

```diff
@@ -39,13 +39,16 @@ client.once("ready", () => {
 	const reminders = db.prepare("SELECT * from reminders").all();
 	const currentTime = Date.now();
 	reminders.forEach(async (reminder) => {
-		const userToRemind = await client.users.fetch(reminder.user_id, true);
-		const channelToPost = await client.channels.fetch(reminder.channel_id, true);
+		const userToRemind = await client.users.fetch(reminder.user_id);
+		const channelToPost = await client.channels.fetch(reminder.channel_id);
 		const timeToRun = reminder.end_time - currentTime;
 		const removeReminder = db.prepare("DELETE FROM reminders WHERE id = (?)");
+		const reminderEmbed = new Discord.MessageEmbed()
+			.setColor("#36393f")
+			.setDescription(`⏰ ${reminder.message}`)
 
 		const onCompletion = () => {
-			channelToPost.send(`💖 **${userToRemind.toString()}, here's your reminder: ${reminder.message}.**`, { disableMentions: "everyone" });
+			channelToPost.send(`💖 **${userToRemind.toString()}**, here's your reminder:`, { embed: reminderEmbed });
```

Comment:
do we need to disableMentions: everyone?

Reply:
Nope, because we're now sending it in an embed. Embed fields don't mention users (at the time of writing)! :nerd_face: